index.js

```diff
@@ -12,7 +12,7 @@ module.exports = (response) => {
 	const contentType = response.headers['content-type'];
 	const isHtml = contentType && contentType.split(';')[0] === 'text/html';
 	if (isHtml) {
-		return getPhantomHtml(response.request.href);
+		return getPhantomHtml(response.request.href, response.request.proxy);
```

Comment:
What happens in there is no `proxy` in request?

Reply:
Yes, you're right. I have to admit, I wasn't really able to check if it still works without proxy in options. Let me do some test, make some changes and PR back. Thanks.